src/teardown_lambda/teardown.py

```diff
@@ -0,0 +1,71 @@
+import json
+import os
+
+from common import common, database
+
+logger = None
+
+# Copy tables
+copy_queries = [
+    # Claimant
+    """
+    CREATE TABLE IF NOT EXISTS claimant_stage LIKE claimant_old;
+    INSERT INTO claimant_stage (id, data) SELECT id, data from claimant_old;
```

Comment:
I would probably create a variable named `sql_statement` with `CREATE TABLE IF NOT EXISTS [db_name]_stage LIKE [db_name]_old;
    INSERT INTO [db_name]_stage (id, data) SELECT id, data from [db_name]_old;` in and then when creating this array do something like `sql_statement.replace("[db_name]", "claimant")` and so on for the three, to allow one place to change the statement in the future.